app/controllers/attendances_controller.rb

```diff
@@ -1,4 +1,40 @@
 class AttendancesController < ApplicationController
   def index
   end
+
+  def new
+    @attendance = Attendance.new
+    @company = Company.where(disable: false)
+  end
+
+  def create
```

Comment:
This method has too many lines. Usually, a method with more than 10 lines of code is considered too long in Ruby. You could try breaking it into smaller methods.

Reply:
Done